draft-ietf-tls-dtls-rrc.md

```diff
@@ -528,7 +518,7 @@ suitable value.
 # Example
 
 In the example TLS 1.3 handshake shown in {{fig-handshake}}, a client
-and a server successfully negotiate support for CID as well as the RRC
+and a server successfully negotiate support for both CID and the RRC
```

Comment:
TLS? CID is not supported for TLS, instead it's DTLS only, or?

Reply:
Fixed in [a2c7fd8](https://github.com/tlswg/dtls-rrc/pull/63/commits/a2c7fd8d91103cf047fb69ae02d9892593ce5e88)